estimate-web/src/components/UserStoryDiscussion.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import UserStoryCommentEditor from "./UserStoryCommentEditor";
+import userShape from "../reducers/models/userShape";
+
+import "./UserStoryDiscussion.less";
+import workItemShape from "../reducers/models/workItemShape";
+import UserStoryCommentList from "./UserStoryCommentList";
+
+const UserStoryDiscussion = ({ user, workItem }) => (
```

Comment:
This component name could be improved as `UserStoryDiscussion` and `UserStoryCommentList` doesn't sound that different

Reply:
Any suggestions? I'm struggling to come up with something 😅
I took the terminology roughly from DevOps, they call it that way on the workItem detail page